server/api/auth/login/ldap.post.ts

```diff
@@ -9,13 +9,15 @@ if (!config.jwt_key) {
   throw new Error("jwt_key is not set.");
 }
 
+const ldapEnabled = config.public.ldap_enabled === "true" || config.public.ldap_enabled === true;
```

Comment:
I think the config stuff gets passed through as strings so not sure if we need to compare against the boolean itself.